docs/sdks/languages/go.mdx

```diff
@@ -967,6 +960,9 @@ res, err := client.Kms().Signing().Verify(infisical.KmsVerifyDataOptions{
     <ParamField query="Data" type="string" required>
       The data to verify. Must be a base64 encoded string.
     </ParamField>
+    <ParamField query="IsDigest" type="boolean" optional>
```

Comment:
this is somewhat ambiguous. i think we can be clearer with something like `preDigested`